lib/core/remove-file.ts

```diff
@@ -0,0 +1,70 @@
+import fs from "fs";
+import path from "path";
+import glob from "glob";
+
+import { alerts } from "./alerts";
+import {
+  getScssPathFromTypeDefinition,
+  getTypeDefinitionPath
+} from "../typescript";
+
+/**
+ * Given a single file remove the generated types if they exist
+ *
+ * @param file the SCSS file to generate types for
+ *
+ */
+
+export const removeFile = (file: string): void => {
+  if (fs.existsSync(file)) {
+    return;
+  }
+  const path = getTypeDefinitionPath(file);
+  try {
+    if (fs.existsSync(path)) {
+      fs.unlink(path, err => {
+        if (err) {
+          alerts.error(`Could not remove ${path}`);
+        } else {
+          alerts.success(`[REMOVED TYPES] ${path}`);
+        }
+      });
+    }
+  } catch (error) {
+    alerts.notice(`[NO GENERATED TYPES] for ${file}`);
```

Comment:
Is this the correct error message? I think it's also an `error`?

```suggestion
    alerts.error(`An error occurred removing ${file}:\n${error}`);
```